### DISCUSSION TITLE
would it be useful for others as well if it was fetching the matrix room off the github site?

### DISCUSSION THREAD

**Discussion by basings** [ImranR98/Obtainium#1178] [Ideas]
When I am looking to connect with other jsers of an app, my main method is to use matrix. For this  I open obtainium, go to their github lage and look for the matrix room. Would be be useful to others to have this info on the obtainium screen as well?

**Comment by ImranR98** [comment]
I think that would be out of scope. Not every project will put that info in the same place (assuming they have a matrix room at all).
